### PR TITLE
create reflection parameter from closure

### PR DESCRIPTION
**Pull request by marcosh**
solves #255

### CODE REVIEW

File: src/Reflection/ReflectionParameter.php [hunk 1]
```diff
@@ -99,6 +99,19 @@ public static function createFromClassInstanceAndMethod($instance, $methodName,
     }
 
     /**
+     * Create a reflection of a parameter using a closure
+     *
+     * @param \Closure $closure
+     * @param string $parameterName
+     * @return ReflectionParameter
+     */
+    public static function createFormClosure(\Closure $closure, $parameterName)
```
Comment (asgrim) [inline src/Reflection/ReflectionParameter.php]: * typo in method name `createFromClosure`
 * also please add type declarations (`string $parameterName` and ` : ReflectionParameter`) (I'm working on adding type declarations everywhere else as we're min PHP 7 for next release now)

File: test/unit/Reflection/ReflectionParameterTest.php [hunk 2]
```diff
@@ -49,6 +49,14 @@ public function testCreateFromClassInstanceAndMethod()
         $this->assertSame('index', $parameterInfo->getName());
     }
 
+    public function testCreateFromCallable()
+    {
+        $parameterInfo = ReflectionParameter::createFormClosure(function ($a) {}, 'a');
+
+        $this->assertInstanceOf(ReflectionParameter::class, $parameterInfo);
```
Comment (asgrim) [inline test/unit/Reflection/ReflectionParameterTest.php]: please use `self::assertInstanceOf`

File: test/unit/Reflection/ReflectionParameterTest.php [hunk 3]
```diff
@@ -49,6 +49,14 @@ public function testCreateFromClassInstanceAndMethod()
...
+        $this->assertSame('a', $parameterInfo->getName());
```
Comment (asgrim) [inline test/unit/Reflection/ReflectionParameterTest.php]: please use `self::assertSame`

**Comment by marcosh** [comment]
fixed the minor changes

**Pull request merged**